css/quiz.css

```diff
@@ -0,0 +1,77 @@
+@import url(https://fonts.googleapis.com/css?family=Work+Sans:300,600);
+
+
+body{
+	font-size: 20px;
+  background-color: #000;
+	font-family: 'Work Sans', sans-serif;
+	color: #fff;
+  
+  text-align: center;
+
+}
+h1{
+  font-weight: 700;
+  margin: 0px;
+  padding: 1rem;
+  font-size: 3rem;
+  color: #fff;
+   background-color: #134041; 
+}
+.heading_2{
+  font-size: 1.5rem;
+}
+
+h3{
+  text-align: left;
+  color:#000;
+  font-size: 30px;
+  
+}
+#test{
+  
+  background-color: #f9f9fa;
+  margin:1rem 10rem;
+   padding:3px 10px 3px 100px; 
+   color: #000;
+
+}
+
+ 
+  h4{
+    text-align: left;
+    padding: 10px 0px;
+    width:500px;
+    background-color: #f1f1f1;
+     box-shadow: 0 8px 6px -6px black; 
+     
+    
+    
+    border-radius: 15px;
+  }
+
+button{
+  font-family: 'Work Sans', sans-serif;
+	font-size: 20px;
+  font-weight: 700;
+	background-color:  #49b1ad;
+	color: #fff;
+	border: 0px;
+	border-radius: 3px;
+	padding: 20px;
+	cursor: pointer;
+	margin-bottom: 20px;
+}
+button:hover{
+	background-color: #3cae3c;
+}
+
+
+
+
+
+.container{
```

Comment:
You can include bootstrap and use those styles as in our website we are using bootstrap.